XiEditor/AppDelegate.swift

```diff
@@ -76,8 +110,17 @@ class AppDelegate: NSObject, NSApplicationDelegate, XiClient {
                                                   textColor: theme.foreground)
         }
     }
+    
+    override init() {
+        ValueTransformer.setValueTransformer(
```

Comment:
Nope.  It crashes when you put it there.  IIRC it's because the value transformer is instantiated before applicationWillFinishLaunching is called so it crashes loading the nib cause it can't find it.

Reply:
Okay, good to know.